### ISSUE TITLE
Select component's required="true" property doesn't produce the * mark

### ISSUE THREAD

**Issue by babdullah-at-dxc-dot-com** [dxc-technology/halstack-angular#338]
**Describe the bug**
Select component's required="true" property doesn't produce the * mark

**To Reproduce**
See the example code.

Example:

 ```
<dxc-select label="Power Type" required="true" margin="xxsmall">
 </dxc-select>
```

**Expected behavior**
An astric mark is expected to appear in UI.

**Comment by Jialecl** [comment]
The issue has been fixed and it is available in the next version.

Here is an example:
https://codesandbox.io/s/required-select-c3qcq

It will also be available in the next release.

**Comment by mawelCaballero** [comment]
This fix has been included in the release 3.0.2
https://www.npmjs.com/package/@dxc-technology/halstack-angular/v/3.0.2
